basyx.aasregistry/basyx.aasregistry-client-native/src/main/java/org/eclipse/digitaltwin/basyx/aasregistry/main/client/mapper/AttributeMapper.java

```diff
@@ -146,4 +149,80 @@ public AssetKind mapAssetKind(org.eclipse.digitaltwin.aas4j.v3.model.AssetKind a
 		}
 	}
 
+	/**
+	 * Maps {@link AssetInformation#getSpecificAssetIds()} from AAS4J to AasRegistry client
+	 *
+	 * @param specificAssetIds
+	 * @return the mapped specificAssetIds
+	 */
+	public List<SpecificAssetId> mapSpecificAssetIds(List<org.eclipse.digitaltwin.aas4j.v3.model.SpecificAssetId> specificAssetIds) {
+		if (specificAssetIds == null || specificAssetIds.isEmpty()) {
+			return null;
+		}
+		List<org.eclipse.digitaltwin.basyx.aasregistry.client.model.SpecificAssetId> mappedSpecificAssetIds = new ArrayList<>();
+		for (org.eclipse.digitaltwin.aas4j.v3.model.SpecificAssetId specificAssetId : specificAssetIds) {
+			org.eclipse.digitaltwin.basyx.aasregistry.client.model.SpecificAssetId mappedSpecificAssetId =
```

Comment:
Also, there is no need to write FQN of the SpecificAssetId that belongs to the client.